pylint/reporters/sarif_types.py

```diff
@@ -0,0 +1,134 @@
+"""
```

Comment:
We can create a package and move this to ``pylint/reporters/sarif/types.py`` I guess